ci/tests/rp-flux/test.sh

```diff
@@ -26,7 +26,7 @@ radical-stack
 ret=$?
 echo "--- smoke test $ret"
 
-SID=$(ls -rt | grep rp.session)
+SID=$(ls -rt | grep rp.session | tail -1)
```

Comment:
Nitpick: `tail -1` is a GNU extension - the [POSIX compliant option](https://pubs.opengroup.org/onlinepubs/007904875/utilities/tail.html) would be `tail -n -1` :-D

Reply:
good catch :) I would suggest to apply these changes with some following PR/update

Reply:
Yeah, that's not urgent and I doubt it will cause any trouble...